roles/sap_ha_install_anydb_ibmdb2/meta/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+galaxy_info:
+  namespace: community
+  author: Sean Freeman
+  description: HA for SAP AnyDB - IBM Db2
+  company: Red Hat, Inc.
+  license: Apache-2.0
+  min_ansible_version: 2.9
+  platforms:
+    - name: EL
```

Comment:
In the role code I see that SUSE is also supported, is it ?

Reply:
Yes, `compatibility` for SLES is contained within. Search code for Zypper usage and different when for SLES path.